Makefile

```diff
@@ -113,15 +113,19 @@ terraform-common-apply: terraform-common-init ## make terraform-common-apply
 
 ##@ terraform/monitoring. Deploys grafana, prometheus monitoring on Gov.UK PaaS
 
+.PHONY: monitoring-init
+monitoring-init:
+		$(if $(passcode), , $(error Missing environment variable "passcode"))
+		$(eval export TF_VAR_paas_sso_passcode=$(passcode))
+		terraform init -upgrade=true -input=false terraform/monitoring
```

Comment:
In [TEVA-1534] I added `-reconfigure` to common.
Would line 120 be improved by being 
```
terraform init -upgrade=true -input=false -reconfigure terraform/monitoring
```

By moving out the init to its own action, you seem to have dropped the workspace (which in `monitoring-plan` and `monitoring-apply` was being set with `TF_WORKSPACE=monitoring`

I suggest a line 121, like `terraform-common-init`, of 
```
terraform workspace select monitoring terraform/monitoring
```

Reply:
As agreed with @nickromneydfe , we're not going to manage workspaces here. We will remove them entirely in a separate story to avoid confusion.